src/app/screens/transactionRequest/index.tsx

```diff
@@ -139,6 +139,22 @@ function TransactionRequest() {
     setHasSwitchedAccount(true);
   };
 
+  const handleTxSigningRequest = () => {
+    const stacksTransaction = deserializeTransaction(payload.txHex!);
+    const { hashMode } = stacksTransaction.auth.spendingCondition;
+    const isMultiSigTx = isMultiSig(hashMode);
+    navigate('/confirm-stx-tx', {
+      state: {
+        unsignedTx: payload.txHex,
+        sponosred: payload.sponsored,
+        isBrowserTx: true,
+        tabId,
+        requestToken,
+        isMultiSig: isMultiSigTx,
+      },
+    });
```

Comment:
```suggestion
    const stacksTransaction = deserializeTransaction(payload.txHex!);
    const { hashMode } = stacksTransaction.auth.spendingCondition;
    const isMultiSigTx = isMultiSig(hashMode);
    navigate('/confirm-stx-tx', {
      state: {
        unsignedTx: payload.txHex,
        sponsored: payload.sponsored,
        isBrowserTx: true,
        tabId,
        requestToken,
        isMultiSig: isMultiSigTx,
      },
    });
```

Reply:
@m-aboelenein this wasn't addressed yet

Reply:
i think i addressed both which one do you think i haven't yet?

Reply:
aha ok, let me recheck